client/app/components/Modal/index.jsx

```diff
@@ -145,7 +145,7 @@ export const Modal = (props: Props): Node => {
           id={elementId}
           className={`modalElement ${css.modalElement} ${className || ''}`}
           onClick={toggleOpen}
-          onKeyDown={toggleOpen}
+          onKeyDown={(event) => handleKeyPress(event, 'Enter')}
```

Comment:
This change makes the modal open upon hitting `Enter`, however, if we skip hitting `Enter` and continue to `Tab` through, it will focus on the `modalElement` twice.

This happens with the Notification modal in the signed-in dashboard:

![Oct-02-2022 00-06-01](https://user-images.githubusercontent.com/3010728/193437569-73db34d6-6d22-4f71-9473-eb48df2be4cd.gif)